app/views/posts/show.html.erb

```diff
@@ -1,5 +1,25 @@
-<h1>Post by <%= @user.name %></h1>
-<h2><%= @post.title %></h2>
-<p><%= @post.text %></p>
-<p>Comments: <%= @post.comments_counter %></p>
-<p>Likes: <%= @post.likes_counter %></p>
\ No newline at end of file
+<a href="<%= user_posts_path(@user) %>">
+  <button class="custom-back-btn">⬅</button>
+</a>
+<div class="custom-post-container">
```

Comment:
- [ ] Kindly fix this back button's path as its cuasing an error and also please instantiate the @user variable as well to use it in the views/posts/show page

Reply:
This change has been done well